.env

```diff
@@ -0,0 +1,13 @@
+IMAGE_VERSION=0.0.0
+DOCKERHUB_NAME=yeahshad
+IMAGE_NAME=express-gallery
+
+EXPRESS_CONTAINER_PORT=8989
+EXPRESS_HOST_PORT=8989
+
+POSTGRES_HOST_PORT=5432
+POSTGRES_CONTAINER_PORT=5432
+POSTGRES_HOSTNAME=postgres-primary-db
+POSTGRES_USER=express_gallery_user
+POSTGRES_PASSWORD=password
+POSTGRES_DB=express_gallery_db
```

Comment:
the `.env` file should be in your `.gitignore`. This is why we have a `.env.example` file.